test/helpers/analytics_helpers.rb

```diff
@@ -0,0 +1,22 @@
+module AnalyticsHelpers
+  # GA is noisy in tests becayse all GA calls become console.log so we
+  # don't want it enabled for all tests, use this to selectively turn it on
+  def with_ga_enabled(&block)
+    begin
+      GovukAdminTemplate.configure { |c| c.enable_google_analytics_in_tests = true }
+      block.call
+    ensure
+      GovukAdminTemplate.configure { |c| c.enable_google_analytics_in_tests = false }
+    end
+  end
+
+  def refute_dimension_is_set(dimension)
+    refute_match(/#{Regexp.escape("GOVUKAdmin.setDimension(#{dimension}")}/, page.body)
+  end
+
+  def assert_dimension_is_set(dimension, with_value: nil)
+    dimension_set_js_code = "GOVUKAdmin.setDimension(#{dimension}"
+    dimension_set_js_code += ", \"#{with_value}\")" if with_value.present?
```

Comment:
Same as in https://github.com/alphagov/whitehall/pull/3293, a closing bracket is missing. 

I'm going to repeat what I said there: 

```
So if there's no with_value present, we're missing a closing bracket for the previous line ?

Would a better solution be:
dimension_set_js_code += with_value.present? ? ", \"#{with_value}\")" : ")"
?

It hasn't failed because we do regex matching
```